src/app/core/demo-data/demo-data-initializer.service.ts

```diff
@@ -69,7 +69,12 @@ export class DemoDataInitializerService {
       DemoDataGeneratingProgressDialogComponent,
     );
 
-    await this.demoDataService.publishDemoData();
+    if (window["ndbDemoData"]) {
```

Comment:
the "ndb" prefix (assuming you picked this from the ndb-core repo name?) is super old and a bit out of context nowadays. This was the working name initially but now we are only using "aam".
Maybe I would rename this to just "demoData" here - or "e2eDemoData"?

Reply:
Makes sense, fixed in https://github.com/Aam-Digital/ndb-core/pull/3045/commits/93552204d956d80413e2df23b8d9adc081728969